src/base/bittorrent/torrenthandle.cpp

```diff
@@ -433,17 +434,28 @@ QHash<QString, TrackerInfo> TorrentHandle::trackerInfos() const
 
 void TorrentHandle::addTrackers(const QVector<TrackerEntry> &trackers)
```

Comment:
On a second thought, I think you probably shouldn't make any modifications to this function. This function takes `QVector<TrackerEntry>` parameter and you should set the tiers to it.